src/renderer/dx/ScreenPixelShader.h

```diff
@@ -4,84 +4,170 @@
 const char screenPixelShaderString[] = "";
 #else
 const char screenPixelShaderString[] = R"(
+// In order to run in KodeLife (great Shader IDE and free with some nagging), define the following:
+// #define KODELIFE
+

```

Comment:
The shader can be modified in multiple ways such as how extreme are the bulge and scan lines?